.changeset/many-cheetahs-hear.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/badge': patch
```

Comment:
Since the badge will look different, should it be a minor change?

Reply:
I don't think so - but what do you think?

Reply:
Nvm. UPdated.

Reply:
What made you switch? Honestly, I'm not so sure myself. The documents are not that helpful.

Reply:
Talked it out with Rob and Dave. It's not a bug, so it's not really a patch.